### PR TITLE
Try Travis

### PR DESCRIPTION
**Pull request by szepeviktor**
See: https://github.com/sektioneins/suhosin7/issues/5

### CODE REVIEW

**Comment by szepeviktor** [comment]
But others failed https://travis-ci.org/szepeviktor/suhosin7/builds/118717807
Is it only for amd64 architecture.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+php:
+  - 5.3
+  - 5.4
+  - 5.5
+  - 5.6
+  - 7.0
+
+env:
+  - CC=gcc-4.8 CFLAGS=""
+  - CC=clang   CFLAGS=""
```
Comment (szepeviktor) [inline .travis.yml]: Maybe I haven'§t installed clang. 
https://travis-ci.org/szepeviktor/suhosin7/jobs/118717816#L153

**Comment by szepeviktor** [comment]
After all both 7.0 builds are green: https://travis-ci.org/szepeviktor/suhosin7

I think https://github.com/sektioneins/suhosin7 is not PHP 5.x compatible.

**Comment by szepeviktor** [comment]
Todo:
- x86 32bit+64bit / ARM / ARM64
- PHP with ZTS and without ZTS
- numerous supported PHP versions :ok:
- different compilers (clang + GCC) :ok:
- with and without experimental features
- in combination with some PHP modules, such as opcache, apc or proprietary code encryption modules (ioncube)

**Comment by jvoisin** [comment]
It would be great to have this merged, to quickly spot regressions for merge-requests.

**Comment by szepeviktor** [comment]
> for merge-requests.

To be clear these are caller PR-s here in GitHub.

**Pull request merged**